### PR TITLE
Semantic UI Inputs

### PR DESCRIPTION
**Pull request by ashbaldry**
I've noticed that you've updated shiny.semantic to Fomantic UI, which is amazing 😄. I have been using this a bit with some of the newer components, and was going through the [form](https://fomantic-ui.com/collections/form) section and hoping these inputs help.

- `uicalendar` is the [calendar](https://fomantic-ui.com/modules/calendar) module for Formantic UI
- `multiple_radio` is similar to `multiple_checkbox`, but the radio only allows one option, whereas checkbox allows multiple and server receives it as a vector.
- `uitextinput` is a simple text input, but can be updated from the server using shiny's `updateTextInput`.
- `uinumericinput` is like `uitextinput` except on the server side it is read as a number.

I have also made a simple application too containing all of these input types.

Do let me know if I've missed anything, or any changes I've made need tweaking.

### CODE REVIEW

**Comment by dokato** [comment]
Oh man @ashbaldry , that's a really great chunk of work... Big kudos for that! We're now in the process of reviewing the changes and will contact you shortly. At the first glance looks really good.

File: examples/form_inputs/www/shiny-semantic-radiogroup.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+var semanticRadioGroupBinding = new Shiny.InputBinding();
```
Comment (dokato) [inline examples/form_inputs/www/shiny-semantic-radiogroup.js]: Don't we want to have it in `inst/www`?

File: tests/testthat/test_input.R [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  # empty input
+  expect_error(uinumericinput())
+  # text input
+  expect_error(uinumericinput("number input", "Text input"))
+  # number input
+  si_str <- as.character(uinumericinput("number_input", 20))
+  expect_equal("<input id=\"number_input\" value=\"20\" type=\"number\"/>", si_str)
+  # all parameters
+  expect_is(uinumericinput("number_input", 20, min = 10, max = 40, step = 1), "shiny.tag")
+})
```
Comment (dokato) [inline tests/testthat/test_input.R]: Thanks for some tests

File: examples/form_inputs/app.R [hunk 3]
```diff
@@ -0,0 +1,147 @@
+library(shiny)
+library(shiny.semantic)
+
+ui <- shinyUI(
+  semanticPage(
+    tags$script(src = "shiny-semantic-radiogroup.js"),
+    suppressDependencies("bootstrap"),
```
Comment (dokato) [inline examples/form_inputs/app.R]: is this needed here?

File: examples/form_inputs/www/shiny-semantic-radiogroup.js (hunk 1 above)
Comment (ashbaldry) [reply examples/form_inputs/www/shiny-semantic-radiogroup.js]: Oops...left that file in by accident, was testing it out but can use the `shiny-input-radiogroup` class instead.

File: examples/form_inputs/app.R (hunk 3 above)
Comment (ashbaldry) [reply examples/form_inputs/app.R]: Nah, that can be removed with the JS file

**Pull request merged**